Doc/distutils/sourcedist.rst

```diff
@@ -95,8 +95,9 @@ source distribution:
   distributions, but in the future there will be a standard for testing Python
   module distributions)
 
-* :file:`README.txt` (or :file:`README`), :file:`setup.py` (or whatever  you
-  called your setup script), and :file:`setup.cfg`
+* Any of the standard README files (:file:`README`, :file:`README.txt`,
+  or :file:`README.rst`), :file:`setup.py` (or whatever you called your setup
+  script), and :file:`setup.cfg`.
```

Comment:
Also, there should be an issue for this on bugs.python.org and the PR title should reference that issue.  It looks like this is a duplicate of bpo-11913 which was originally closed as "wont fix".  It could be reopened and used to track this.